resources/conf/log4j2.properties

```diff
@@ -14,7 +14,7 @@ appender.ENFORCER_LOGFILE.name = ENFORCER_LOGFILE
 appender.ENFORCER_LOGFILE.fileName = logs/enforcer.log
 appender.ENFORCER_LOGFILE.filePattern = /logs/enforcer-%d{MM-dd-yyyy}.log
 appender.ENFORCER_LOGFILE.layout.type = PatternLayout
-appender.ENFORCER_LOGFILE.layout.pattern = [%d][%X{traceId}] %5p {%c} - %m%ex%n
+appender.ENFORCER_LOGFILE.layout.pattern = [%d][%X{traceId}] %5p {%c} - %m%ex %x%n
```

Comment:
can include stack details after traceId

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2679/commits/356dd76f69e350e032432200814911fd84269ee1